### ISSUE TITLE
1.19 mypet cannot be saved

### ISSUE THREAD

**Issue by axin0452** [MyPetORG/MyPet#1549]
# Bug report

* [ √] I have checked other issues to make sure this is not a duplicate.
I tried 3 times and couldn't save the pet

## Describe the bug
In the 1.19paper server, I successfully captured warden, and there was no problem in the game, but when I stopped and closed the server and started the server again, it showed that I had no pet.
If a player captures warden and shuts down the server after going offline, warden will become a wild monster, and it will show that there is no pet when he goes online again.This problem has a certain probability.
It has been observed that there seems to be a problem with the level storage of mypet. After restarting the server several times, the level will be cleared each time.

There is a bug report when shutting down the server

```
[04:01:57] [Server thread/INFO]: [de.Keyle.MyPet.MyPetPlugin] [[0;32;22mMyPet[m] [mDisabling MyPet v3.12[m
[04:01:57] [Server thread/ERROR]: Error occurred while disabling MyPet v3.12 (Is it up to date?)
org.bukkit.plugin.IllegalPluginAccessException: Plugin attempted to register task while disabled
	at org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftScheduler.validate(CraftScheduler.java:552) ~[paper-1.19.jar:git-Paper-20]
	at org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftScheduler.runTaskTimer(CraftScheduler.java:227) ~[paper-1.19.jar:git-Paper-20]
	at org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftScheduler.runTaskTimer(CraftScheduler.java:212) ~[paper-1.19.jar:git-Paper-20]
	at org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftScheduler.runTaskLater(CraftScheduler.java:176) ~[paper-1.19.jar:git-Paper-20]
	at de.Keyle.MyPet.compat.v1_19_R1.entity.types.EntityMyWarden.a(EntityMyWarden.java:109) ~[MyPet-3.12-SNAPSHOT-B1693.jar:?]
	at net.minecraft.world.entity.Entity.discard(Entity.java:652) ~[paper-1.19.jar:git-Paper-20]
	at de.Keyle.MyPet.compat.v1_19_R1.entity.CraftMyPet.removeEntity(CraftMyPet.java:101) ~[MyPet-3.12-SNAPSHOT-B1693.jar:?]
	at de.Keyle.MyPet.entity.MyPet.removePet(MyPet.java:578) ~[MyPet-3.12-SNAPSHOT-B1693.jar:?]
	at de.Keyle.MyPet.entity.MyPet.removePet(MyPet.java:587) ~[MyPet-3.12-SNAPSHOT-B1693.jar:?]
	at de.Keyle.MyPet.MyPetPlugin.onDisable(MyPetPlugin.java:102) ~[MyPet-3.12-SNAPSHOT-B1693.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:266) ~[paper-api-1.19-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.java.JavaPluginLoader.disablePlugin(JavaPluginLoader.java:399) ~[paper-api-1.19-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.disablePlugin(SimplePluginManager.java:573) ~[paper-api-1.19-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.disablePlugins(SimplePluginManager.java:550) ~[paper-api-1.19-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_19_R1.CraftServer.disablePlugins(CraftServer.java:488) ~[paper-1.19.jar:git-Paper-20]
	at net.minecraft.server.MinecraftServer.stopServer(MinecraftServer.java:929) ~[paper-1.19.jar:git-Paper-20]
	at net.minecraft.server.dedicated.DedicatedServer.stopServer(DedicatedServer.java:781) ~[paper-1.19.jar:git-Paper-20]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1229) ~[paper-1.19.jar:git-Paper-20]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:302) ~[paper-1.19.jar:git-Paper-20]
	at java.lang.Thread.run(Unknown Source) ~[?:?]
```

MyPet-3.12-SNAPSHOT-B1693
paper1.19-20
[ticket.zip](https://github.com/MyPetORG/MyPet/files/8922118/ticket.zip)

**Comment by Jakllp** [comment]
Interesting - I know why this is.

Just for confirmation for me tho:
If you do `/petsa` or `/petstore` before restarting the server and wait for the animation to end.
If it gets saved I'm right, if not further investigation is needed

**Comment by axin0452** [comment]
@Jakllp 
I don't really understand what it means, do you have a discord, my English is not good, maybe I can't speak clearly

**Comment by Jakllp** [comment]
Your description of the bug together with the log you provided were perfect.

I was just asking you if you could test something for me as that might speed things up later when I get to fixing this bug.

But it's fine.

I believe I knowwhy the bug is happening and will fix it soon

**Comment by axin0452** [comment]
I just tried the commands /petsa and /petstore and /petcall and /petswitch, they are called by the command and the exp is correct, but when I stop the server my warden is gone, but when I test the zombie it is still there and exp is the correct value
@Jakllp

**Comment by Jakllp** [comment]
Fixed in `3.12-SNAPSHOT-B1694`
with https://github.com/MyPetORG/MyPet/commit/6d1de892aff04a0412505685bf21a9d0358fdc5c
